<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Deposit amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_start_summ">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>1000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000000.000000000000000</double>
          </property>
          <property name="value">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Term (in months)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_months">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Interest Rate (%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_percent">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Tax rate (%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_tax_rate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Compounding periods</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_payments">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="currentText">
           <string notr="true">Monthly</string>
          </property>
          <item>
           <property name="text">
            <string>Monthly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Quarterly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Semi-annually</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Annually</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_cap">
        <property name="styleSheet">
         <string notr="true">QCheckBox {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
        </property>
        <property name="text">
         <string>Capitalization</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Replenishment / withdrawal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_add">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: #4F4C79;
	background-color:  #DEDDFC;
	border: 1px transparent;
	border-radius: 14px;
	font: Rubik;
	font-size: 14px;
}

QPushButton:hover {
	background-color: #dfd3f2;
}

QPushButton:pressed {
	background-color: #7c50b4;
}</string>
          </property>
          <property name="text">
           <string>Add +</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_table">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget_deposit">
        <property name="minimumSize">
         <size>
          <width>410</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>Month</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Replenishment</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Withdrawal</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>279</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #E6EAFF;
	background-color:  #5B5B8F;
	border: 1px transparent;
	border-radius: 14px;
}

QPushButton:hover {
	background-color: #786097;
}

QPushButton:pressed {
	background-color: #4F4F7D;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #4F4C79;
	background-color:  #DEDDFC;
	border: 1px transparent;
	border-radius: 14px;
	font: Rubik;
	font-size: 14px;
}

QPushButton:hover {
	background-color: #dfd3f2;
}

QPushButton:pressed {
	background-color: #7c50b4;
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>22</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">QLabel {	
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Total amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_summ_end">
          <property name="styleSheet">
           <string notr="true">QLabel {	
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Interest Earned</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_summ_percent">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="text">
           <string>Tax amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_summ_tax">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: 1px transparent;
	color: #4F4C79;
	background-color: #F0EFFF;
	font-family: Rubik;
	font-size: 14pt;
}</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_error">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #5B5B8F;
	font-size: 14px;
}</string>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
